simplefs/simplefs.go

```diff
@@ -1771,6 +1771,18 @@ func (k *SimpleFS) SimpleFSSyncStatus(ctx context.Context) (keybase1.FSSyncStatu
 		return keybase1.FSSyncStatus{}, nil
 	}
 
+	var syncingPaths []string
+	if filter == keybase1.ListFilter_NO_FILTER {
+		syncingPaths = status.UnflushedPaths
+	} else {
+		for _, p := range status.UnflushedPaths {
+			if isFiltered(filter, p) {
+				continue
+			}
+			syncingPaths = append(syncingPaths, p)
+		}
+	}
+
 	k.log.CDebugf(ctx, "Sending sync status response with %d syncing bytes",
 		status.UnflushedBytes)
 	return keybase1.FSSyncStatus{
```

Comment:
Looks like you forgot to return `syncingPaths` here.

Reply:
whoops good catch